src/housing/modeling/score.py

```diff
@@ -2,7 +2,7 @@
 import os
 import pickle as pkl
 
-from housing.preparation import data_utils
+# from housing.preparation import data_utils
```

Comment:
pls. remove commented code that is no longer required (review elsewhere in this file as well)